apps/src/netsim/NetSimTable.js

```diff
@@ -133,11 +172,11 @@ NetSimTable.prototype.readAll = function (callback) {
  */
 NetSimTable.prototype.makeThrottledRefresh_ = function (waitMs) {
   return _.throttle(function () {
-    this.clientApi_.all(function (err, data) {
-      if (err === null) {
-        this.fullCacheUpdate_(data);
-      }
-    }.bind(this));
+    if (this.useIncrementalRefresh_) {
```

Comment:
Yes, I think that's right.   This makes is easier and less error-prone, and limits the number of places in the code that need to know what kind of refresh is appropriate for a particular table.  (Otherwise code might accidentally call refreshFromId on a table that wasn't designed for it. Also, we would need to update multiple places in the code if we changed what kind of refresh was appropriate for the table, vs. just the construction.)